tests/Areas/Server/UnitTests/AppConfigJsonSchemaTests.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
It feels odd to have an AppConfig-specific test at the /Server level. I don't mind keeping this here for now, but I think we can reuse this and make it more generalized so we can verify other server commands.